src/databricks/labs/ucx/source_code/folders.py

```diff
@@ -0,0 +1,59 @@
+from __future__ import annotations
+
+from pathlib import Path
+from collections.abc import Iterable
+
+from databricks.labs.ucx.source_code.base import is_a_notebook
+from databricks.labs.ucx.source_code.files import FileLoader
+from databricks.labs.ucx.source_code.graph import SourceContainer, DependencyGraph, DependencyProblem, Dependency
+from databricks.labs.ucx.source_code.notebooks.loaders import NotebookLoader
+from databricks.labs.ucx.source_code.path_lookup import PathLookup
+
+
+class Folder(SourceContainer):
```

Comment:
Coming from `linters/files.py` but not per se linting and splitting from `files.py` to match `linters/folders.py`